### PR TITLE
Privacy policy admin tab

### PR DESCRIPTION
**Pull request by andysmitharng**

### CODE REVIEW

File: app/admin/terms_and_conditions.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@ def action_methods
   index do
     selectable_column
     id_column
-    column :updated_at
+    column :updated_atit
```
Comment (JGYaniv) [inline app/admin/terms_and_conditions.rb]:
```suggestion
    column :updated_at
```

File: config/database.yml [hunk 2]
```diff
@@ -11,11 +11,11 @@
...
-  pool: <%= ENV.fetch("DB_POOL_SIZE") { ENV.fetch("RAILS_MAX_THREADS") { 5 } * 1.2 } %>
-  username: <%= ENV['DB_USERNAME'] %>
-  password: <%= ENV['DB_PASSWORD'] %>
-  host: <%= ENV['DB_HOST'] %>
-  port: <%= ENV['DB_PORT'] || "5432" %>
+  encoding: utf8
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  username: postgres
+  password:
+  host: localhost
```
Comment (JGYaniv) [inline config/database.yml]: Revert these changes please, we don't want it in version control as part of this PR & it will result in a conflict with some changes I am working on.

File: app/models/privacy_policy.rb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+class PrivacyPolicy < ActiveRecord::Base  
+  self.table_name = 'privacy_policy'
```
Comment (JGYaniv) [inline app/models/privacy_policy.rb:4]: We probably want to require that a privacy policy at a minimum includes some text in both languages.

```suggestion
  validates :description, :spanish_description, presence: true

  self.table_name = 'privacy_policy'
```

File: db/migrate/20210829163738_add_privacy_policy.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+class AddPrivacyPolicy < ActiveRecord::Migration[5.2]
+  def change
+    create_table :privacy_policy do |t|
+      t.text :description
+      t.text :spanish_description
```
Comment (JGYaniv) [inline db/migrate/20210829163738_add_privacy_policy.rb]:
```suggestion
      t.text :description, null: false
      t.text :spanish_description, null: false
```

**Review by JGYaniv — CHANGES_REQUESTED** [review]
Looks good, but you have a typo and should add the model/db level validations for the privacy policy table.

File: app/admin/terms_and_conditions.rb (hunk 1 above)
Comment (andysmitharng) [reply app/admin/terms_and_conditions.rb]: done

File: config/database.yml (hunk 2 above)
Comment (andysmitharng) [reply config/database.yml]: done

File: app/models/privacy_policy.rb (hunk 3 above)
Comment (andysmitharng) [reply app/models/privacy_policy.rb:4]: done

File: db/migrate/20210829163738_add_privacy_policy.rb (hunk 4 above)
Comment (andysmitharng) [reply db/migrate/20210829163738_add_privacy_policy.rb]: done

**Review by JGYaniv — APPROVED** [review]
Changes look good!
